version.mk

```diff
@@ -48,3 +48,25 @@ helm-version:
 		sed -i'.bak' -e "s_^\\.version:\ .*_.version: \\&version \"$${VERSION}\"_g" examples/chart/$${CHART}/Chart.yaml || exit 1; \
 		rm -f examples/chart/$${CHART}/Chart.yaml.bak; \
 	done
+
+# This awk is a little hacky - xml is not line-based - but this avoids adding
+# another tool (such as xmlstarlet or plutil) as a requirement of this repo.
+# This works well enough for our limited XML.
+# CFBundleShortVersionString must be only one, two or three dot separated
+# positive integers. If the version is not that (i.e. has a semver prerelease tag),
+# set the version number to just "1.0", as it has been until now.
+define plist_replace_awk
+BEGIN { if (match(version, /^[0-9]+(\.[0-9]+(\.[0-9]+)?)?$$/) == 0) { version = "1.0" } }
+/<key>CFBundleShortVersionString<\/key>/ { replace = 1 }
```

Comment:
Should we update CFBundleVersion too? I'm no expert, but it seems using the same number for both is alright.

https://stackoverflow.com/a/31921249

Reply:
I was not going to because it appears to be for internal purposes, and we don't use it internally. But even though the recommendations on that stackoverflow post vary, the suggestion most often made is to keep them the same. So, sure I'll do that.